lib/bundler/cli/gem.rb

```diff
@@ -162,5 +163,12 @@ def ask_and_set_test_framework
       test_framework
     end
 
+    def bundler_dependency_version
+      major, minor, *etc = Bundler::VERSION.split(".")
+      version = [major, minor]
+      version << etc.last if etc.last =~ /[a-z]/i
+      version.join(".")
```

Comment:
Hmm... `approximate_recommendation` is only available with RG 2.0+. Previously, it had a different name: https://github.com/rubygems/rubygems/commit/f5035c61750b8a5af376dc30a1ed74e87b0152f3

I could monkey patch in a method alias if it's not already defined, but it probably isn't worth it just for this. I'll use a hybrid of my original code and your change.